Added code contribution guide for linting (#145)
klaytn/caver-js: CONTRIBUTING.md

- https://www.w3schools.com/js/js_conventions.asp
- https://google.github.io/styleguide/jsguide.html

> caver-js uses eslint and prettier to prevent errors and to unify code format.

Reviewer: ```suggestion
caver-js uses eslint with prettier. The linting rules inherit from airbnb.
```

> If you have written code to contribute to caver-js, run the `npm run lint` script to see the results of linting.

Reviewer: ```suggestion
Before you submit a pull request, please run `npm run lint` and correct any errors.
```

> If there is an error in the result of linting, you must fix it.

Reviewer: ```suggestion
```

> Otherwise, the Circle CI process will fail.

Reviewer: ```suggestion
Otherwise, the test on CircleCI will fail, and this prevents merging.
```
